apps/back-end/package-lock.json

```diff
@@ -0,0 +1,2327 @@
+{
```

Comment:
There shouldn't be a package-lock.json here, just one in the root directory which npm workspaces manages

Reply:
Ditto, see above.

Reply:
This file can be generated by the package.json if we need to separate the project out, but I think it will be confusing for development and interfere with npm workspaces if we keep it here whilst in a monorepo